pylot/component_creator.py

```diff
@@ -268,6 +269,12 @@ def add_obstacle_tracking(center_camera_stream,
         obstacles_tracking_stream = \
             pylot.operator_creator.add_perfect_tracking(
                 ground_obstacles_stream, can_bus_stream)
+
+    if FLAGS.evaluate_obstacle_tracking:
+        pylot.operator_creator.add_tracking_evaluation(
```

Comment:
I would add a perfect detector operator here, instead of returning perfect_obstacles_stream in the other method.